src/quark-core/src/QuarkScript.sol

```diff
@@ -156,10 +156,6 @@ abstract contract QuarkScript {
         bytes32 submissionToken = getActiveSubmissionToken();
         uint256 n;
 
-        if (submissionToken == QuarkNonceManagerMetadata.EXHAUSTED) {
```

Comment:
See above line 151:

> `Note: this may not be accurate after any nested calls from a script`

Is that statement still accurate?

Reply:
No, this statement no longer holds. I actually removed this comment in another PR yesterday. See: https://github.com/compound-finance/quark/commit/5e24a38e2d21084e82ede16dd666973924a210d3#diff-a219685a8ac1b6d53f4feb7ad9ab76351cf5f4dd68a30022fc18a4a9ad65467fL151